### ISSUE TITLE
Feature request: Add the possibility to find out if an organisation has an agreement with Vipps

### ISSUE THREAD

**Issue by EirikFinvold** [vippsas/vipps-partner-api#35]
Today we can get MSN's connected to us as a partner which is fine. However, a product order "prefill" won't be handled by you if they don't have an agreement, so if we are going to onboard a new client it would be great to know if they have an agreement or not before ordering a product.
A simple boolean response would be enough from a technical perspective.

**Comment by cloveras** [comment]
The goal is to handle both cases:
1. If the merchant does not have an agreement (merchant agreement, MA), the merchant will be guided through the process as part of the product order (PO)
2. If the merchant does have a MA, the merchant only needs to complete the PO.

We don't think a simple lookup service ("does this merchant have Vipps") is a good idea.

A partner can already check if a merchant has one or more sale units (MSNs) with that partner: https://github.com/vippsas/vipps-partner-api/blob/main/vipps-partner-api.md#get-information-about-a-merchant-based-on-organization-number

**Comment by EirikFinvold** [comment]
Sure, that would be a good way of solving this. 

Todays endpoint where we can get merchant info is quite useless from a perspective where we get a new customer that we want to onboard. The result will always be false.
However, it's quite useful AFTER we have submitted a Product Order "prefill" to get correct info once you have completed the onboarding on your side.

**Comment by cloveras** [comment]
Yes, we have more functionality we want to add to the Partner API. SO, the goal is as described above, but I can not yet give an ETA of that.

There is a lot of "ground work" that must be done before we can make it visible in the Partner API, but it is very high on one wish/priority list.
